N/nng/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nng"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/nanomsg/nng.git", "169221da8d53b2ca4fda76f894bee8505887a7c6"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd nng
+mkdir build
+cd build
+cmake -G Ninja ..
+ninja
+export DESTDIR=$prefix && ninja install
```

Comment:
Please read the documentation: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds.  You must pass the toolchain file.  Also, no need to patch the CMake file, you can set variables from the command line
```suggestion
cd $WORKSPACE/srcdir/nng
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
make -j${nproc}
make install
```

Reply:
Ah yes, these seem like the correct rocks to smash together :) Thanks for the help!